### PR TITLE
finished assignment

### PR DESCRIPTION
**Pull request by davidcspence3**

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
...
-
-//  action 3:
- 
+//  Noun:       cloud
+//  action 1:   the cloud forms
+cloud.form();
+//  action 2:   the cloud rains
+cloud.rain();
+//  action 3:   the cloud dissipates
+cloud.dissipate();
```
Comment (matkatmusic) [inline main.cpp]: As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
**replace** the highlighted noun/action as follows:
- [x] come up with a new noun and 3 actions
- [x] pick a noun that needs `camelCase` for the name
- [x] pick actions that need `camelCase` for their name.

**Review by matkatmusic — CHANGES_REQUESTED** [review]
In addition to the requestion changes below, some of your nouns are indented a space and need to be lined up.
- [ ] revise

**Comment by davidcspence3** [comment]
fixed indentation and added camelCase noun and actions.

File: main.cpp [hunk 2]
```diff
@@ -131,7 +131,14 @@ sun.set();
...
 sun.explode();
-
+//  11)
+//  Noun:       coffeeMachine
+//  action 1:   the coffeeMachine turns one
+coffeeMachine.turnON();
+//  action 2:   the coffeeMachine timer is set
+coffeeMachine.setTimer();
+//  action 3:   the coffeeMachine starts to brew
+coffeeMachine.beginBrew();
```
Comment (matkatmusic) [inline main.cpp]: please **RE-READ** the revisions I originally requested.
- [x] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]
try again!

File: main.cpp (hunk 1 above)
Comment (davidcspence3) [reply main.cpp]: Reread instructions, hopefully corrected.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**